src/m365/aad/commands/app/app-add.ts

```diff
@@ -567,6 +588,28 @@ class AadAppAddCommand extends GraphCommand {
         value: password.secretText
       }));
   }
+  
+  private getCertificateBase64Encoded(args: CommandArgs, logger: Logger): Promise<string | undefined> {
+    if (args.options.certificateBase64Encoded) {
+      return Promise.resolve(args.options.certificateBase64Encoded);
+    }
+    
+    if (fs.existsSync(args.options.certificateFile as string)) {
```

Comment:
We can simplify this code by moving this logic to the `validate` method and assuming here that the file exists